### ISSUE TITLE
Add language parameters to contact us page

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1230]
As a user, or staff person I need to language parameters on the contact page so that I can easily identify the url of either the English or French page. 

WHY are we building?
The contact us page does not have language parameters in the url, unlike other pages such as 
https://documentation.notification.canada.ca/en/
https://documentation.notification.canada.ca/fr/

WHAT are we building?
Language parameters on the url of the contact us page

VALUE created by our solution
Notify staff can easily add hyperlinks to the contact us page from templates or other parts of the site, without having to add the language parameters to the url first.
Screenreaders may also benefit from this.

## Documentation and Artifacts

Good docs, figma mockups, ADRs, screenshots etc.

[Policy Analysis for Bilingual URLs](https://docs.google.com/document/d/1vXH6jNAefbIZrVjVYR4dJRGH891IasfRvce-h91-ebA/edit#heading=h.1042zzbl21j0)

## Acceptance Criteria
When anyone checks the url of the contact us page, the language of the page is indicated. 

- [ ] Generate appropriate log messages so that executions of this feature can be tracked
- [ ] Can misuse of this feature cause harm? If yes, create an alert
- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)

* A11y
* Bilingualism

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (Ping Adrianne to add stuff here or find stuff yourself)

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by andrewleith** [comment]
The admin site doesn't display the current language in the route, but you can append `?lang=en` or `?lang=fr` to any page in order to directly link to a specific language.

Do we think we are failing a specific guideline because of this?  It would be good to understand the benefit of making this change before moving forward as it involves changing how the admin site routing works, and may require a good amount of testing.

**Comment by jzbahrai** [comment]
@YedidaZalik do we know what specific guideline we are failing?

**Comment by YedidaZalik** [comment]
WCAG 2.0 Level AA 3.1.2 Language of Parts
We should identify anything in a different language than the rest of the page.

**Comment by mtoutloff** [comment]
Policy Analysis for requirements for bilingual URLs: https://docs.google.com/document/d/1vXH6jNAefbIZrVjVYR4dJRGH891IasfRvce-h91-ebA/edit

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1230?planning-poker) with Zenhub @amazingphilippe @andrewleith
